rosidl_runtime_c/include/rosidl_runtime_c/service_type_support_struct.h

```diff
@@ -81,12 +82,18 @@ struct rosidl_service_type_support_t
   const void * data;
   /// Pointer to the service type support handler function
   rosidl_service_typesupport_handle_function func;
+  /// Service request message typesupport
+  const rosidl_message_type_support_t * request_typesupport;
```

Comment:
Key change callout: these `rosidl_message_type_support` instances were not available in all contexts. `rosidl_typesupport_fastrtps` exposed them in its ServiceMembers, but `rosidl_typesupport_introspection` only exposes nested MessageMembers with no access to the message typesupport.